Orange/tests/test_pca.py

```diff
@@ -84,6 +84,22 @@ def __ipca_test_helper(self, data, n_com, min_xpl_var):
         pc1_ipca = pca_model.partial_fit(data[1::2]).components_[0]
         self.assertAlmostEqual(abs(pc1_ipca.dot(pc1_pca)), 1, 4)
 
+    def test_truncated_svd(self):
+        data = self.ionosphere
+        self.__truncated_svd_test_helper(data, n_com=3, min_xpl_var=0.5)
+        self.__truncated_svd_test_helper(data, n_com=10, min_xpl_var=0.7)
+        self.__truncated_svd_test_helper(data, n_com=31, min_xpl_var=0.99)
+
+    def __truncated_svd_test_helper(self, data, n_com, min_xpl_var):
```

Comment:
It's actually ok to use full, _expressive_ variable names where common abbreviations don't exist (e.g. `n_` and `min_` are ok).